apps/frontend/src/app/pages/company/create-profile/create-profile.component.ts

```diff
@@ -0,0 +1,101 @@
+import { Component } from '@angular/core';
+import { InputComponent } from '../../../shared/components/input/input.component';
+import { FormFieldComponent } from '../../../shared/components/form-field/form-field.component';
+import {
+  FormBuilder,
+  FormsModule,
+  ReactiveFormsModule,
+  Validators,
+} from '@angular/forms';
+import { NgClass } from '@angular/common';
+import { BackButtonComponent } from '../../../shared/components/back-button/back-button.component';
+import { ToastService } from '../../../services/toast/toast.service';
+
+@Component({
+  selector: 'wagademy-create-profile',
+  standalone: true,
+  imports: [
+    InputComponent,
+    FormFieldComponent,
+    ReactiveFormsModule,
+    NgClass,
+    FormsModule,
+    BackButtonComponent,
+  ],
+  templateUrl: './create-profile.component.html',
+  styleUrl: './create-profile.component.scss',
+})
+export class CreateProfileComponent {
+  profilePhotoFile!: File;
+  profilePhoto!: string;
+  form = this.fb.group({
+    companyName: ['', Validators.required],
+    areaOfExpertise: ['', Validators.required],
+    description: ['', Validators.required],
+    newWord: [''],
```

Comment:
use this instead of "newWord" and also here we have some validation
whatIsTheCompanyLookingFor: [
      this.fb.array<string>(
        [],
        [Validators.maxLength(10), Validators.minLength(1)]
      ),
    ],